managed/models/database.go

```diff
@@ -1168,6 +1168,10 @@ var databaseSchema = [][]string{
 	114: {
 		`ALTER TABLE agents ADD COLUMN environment_variables TEXT`,
 	},
+	115: {
```

Comment:
```suggestion
	120: {
```
To avoid conflicts when rebasing. We will change it again before merging this PR.